src/components/BasicInfo.tsx

```diff
@@ -15,12 +15,16 @@ export const BasicInfo = ({ pet }: any) => {
     >
       <div
         className="rounded-xl shadow-md aspect-auto overflow-hidden"
-        style={{ gridColumn: '1/2' }}
+        style={{ gridColumn: '1/2', maxWidth: '200px' }}
```

Comment:
I wonder why do we need maxWidth here, it messes up with the view on big screens...